### PR TITLE
SPM support for Fairmatic SDK

### PR DESCRIPTION
**Pull request by SagarSDagdu**
## What does this PR do
Enable client apps to integrate the Fairmatic iOS SDK via SPM

### CODE REVIEW

File: Sources/FairmaticSDKTarget/FairmaticSDKTargetPlaceholder.swift [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//
+//  FairmaticSDKTargetPlaceholder.swift
+//  
+//
+//  Created by Sagar Dagdu on 02/05/24.
```
Comment (mohit-fairmatic) [inline Sources/FairmaticSDKTarget/FairmaticSDKTargetPlaceholder.swift]: we usually omit the Created by header in public files. we can keep Fairmatic copyright header instead

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "FairmaticSDK",
+    platforms: [.iOS(.v13)],
```
Comment (mohit-fairmatic) [inline Package.swift:8]: do we need anything special for iPadOS or is it enough?

File: Sources/FairmaticSDKTarget/FairmaticSDKTargetPlaceholder.swift (hunk 1 above)
Comment (SagarSDagdu) [reply Sources/FairmaticSDKTarget/FairmaticSDKTargetPlaceholder.swift]: That is correct, will change this

Comment (SagarSDagdu) [reply Sources/FairmaticSDKTarget/FairmaticSDKTargetPlaceholder.swift]: Done

File: Package.swift (hunk 2 above)
Comment (SagarSDagdu) [reply Package.swift:8]: iOS is enough, suffices both 👍🏻

**Review by mohit-fairmatic — APPROVED** [review]
+1

**Pull request merged**